kong.conf.default

```diff
@@ -221,6 +221,10 @@
 #anonymous_reports = on          # Send anonymous usage data such as error
                                  # stack traces to help improve Kong.
 
+
+#proxy_server                    # Proxy server defined as a URL. Kong will only use
```

Comment:
```suggestion
#proxy_server =                  # Proxy server defined as a URL. Kong will only use
```

Reply:
leaving a trailing `=` follows the pattern in kong.conf.default, consider this resolved

Reply:
This is what I found: 

https://github.com/Kong/kong/blob/3143aa154d7d7bdf1476f13cb0136527685e259e/kong.conf.default#L268-L318
https://github.com/Kong/kong/blob/3143aa154d7d7bdf1476f13cb0136527685e259e/kong.conf.default#L658-L662